examples/randomuser-sqlite.py

```
@@ -1,35 +1,36 @@
-import json # https://docs.python.org/3/library/json.html
-import requests # https://github.com/kennethreitz/requests
-import records # https://github.com/kennethreitz/records
+import json
+import requests
+import records
 
-# randomuser.me generates random 'user' data (name, email, addr, phone number, etc)
-r = requests.get('http://api.randomuser.me/0.6/?nat=us&results=10')
-j = r.json()['results']
+# Fetch random user data from randomuser.me API
+response = requests.get('http://api.randomuser.me/0.6/?nat=us&results=10')
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)